src/js/demo.js

```diff
@@ -14,8 +14,8 @@ new Vue({
 		rules: {
 			url: 'url',
 			dateISO: 'dateISO',
-			integer: 'integer',
-			numeric: 'numeric',
+			integer: 'integer,min,3,max,10',
```

Comment:
oh yes, I just stroke me that this `integer,min,3,max,10` is not really a good approach. it also may result evaluate method in errors in this block I guess
	```
	let index = 0;
		for (let method in methods) {
			//if the rule required exits and there was an error, the
			//stack errors method is called to keep track of rules
			//that did not pass the validation.
			if (this[methods[method]] && ! this[methods[method]](value, methods[index+1])) {
```